auxprogs/joingenes/Makefile

```
@@ -1,20 +1,20 @@
 CC=g++
-CFLAGS=-Wall -std=gnu++0x
+CXXFLAGS+=-Wall -std=gnu++0x
 
 all: joingenes
 
 joingenes: joingenes.o jg_transcript.o jg_ios.o jg_transcript.h jg_ios.h
-	$(CC) $(CFLAGS) $(CXXFLAGS) $(CPPFLAGS) $(LDFLAGS) joingenes.o jg_transcript.o jg_ios.o -o joingenes
+	$(CC) $(CPPFLAGS) $(CXXFLAGS) joingenes.o jg_transcript.o jg_ios.o -o joingenes $(LDFLAGS)
```

```suggestion
	$(CXX) $(CXXFLAGS) $(CPPFLAGS) $(LDFLAGS) joingenes.o jg_transcript.o jg_ios.o -o joingenes
```
- replace CC by CXX as mentioned before
- why should $(LDFLAGS) stay at the end?